lib/matplotlib/rcsetup.py

```diff
@@ -23,6 +23,8 @@
 from matplotlib.fontconfig_pattern import parse_fontconfig_pattern
 from matplotlib.colors import is_color_like
 
+from cycler import cycler, Cycler
```

Comment:
Ahh nevermind my path problem, I had the path pointing to the directory containing the repo rather than the repo itself...
cycler -> dir
cycler.cycler -> ``cycler.py``
cycler.cycler.cycler -> func